rasa_nlu/training_data/formats/markdown.py

```diff
@@ -81,8 +81,11 @@ def _parse_item(self, line):
                 self.training_examples.append(parsed)
             elif self.current_section == SYNONYM:
                 self._add_synonym(item, self.current_title)
-            else:
+            elif self.current_section == REGEX:
                 self.regex_features.append({"name": self.current_title, "pattern": item})
+            elif self.current_section == LOOKUP:
+                lookup_regex = generate_lookup_regex(item)
```

Comment:
this doesn't seem to append the lookup table to the training data object. if that object is dumped again, it will be missing the json entry for the lookup table

Reply:
mhm so ideally we should be able to do this:
- load from file
- export loaded training data object to file
- have the same file

so yes, if we can move that somewhere else, that would probably fix this.

Reply:
this is important because otherwise the conversation between file format isn't seemless anymore (so converting a markdown file to json format would expand the lookup table into the regex)

Reply:
ok I think what I'll do is have `training_data.lookup_tables` be a list of file paths to lookup tables, the same way it's entered in the training data currently.  Then, in the `RegexFeaturizer` I will just load the files, construct the regex patterns, and load these into `RegexFeaturizer.known_patterns`.  Does this sound like a reasonable approach?

Reply:
yes :+1:

Reply:
@tmbo  ok I pushed some changes to implement this.  1.  Updated the tests 2.  Confirmed that it still retains the lookup table file_name format when dumping markdown or json.  3. Confirmed it works with `train_test` function.